tests/testthat/test-cols_merge.R

```diff
@@ -434,3 +434,60 @@ test_that("the `cols_merge_range()` function works correctly", {
     tbl_html_3 %>% as_raw_html()
   )
 })
+
+
+test_that("the `cols_merge_n_pct()` function works correctly", {
+
+  # Check that specific suggested packages are available
+  check_suggests()
+
+  tbl_n_pct <-
+    dplyr::tribble(
+      ~a, ~b,
+      1,  0.0714,
+      5,  0.3571,
+      0,  0.0,
```

Comment:
Let's add a line for `0, NA` and `NA, 0`

Reply:
Now added.